content/api/ip-pools.apib

```diff
@@ -6,19 +6,50 @@ description: Manage IP pools, which allow you to group and assign dedicated IPs
 
 <banner status="info"><strong>Note</strong>: This endpoint is available for SparkPost customers only. <a href="https://www.sparkpost.com/enterprise-email/">SparkPost Enterprise</a> customers, contact your TAM for details.</banner>
```

Comment:
This endpoint isn't available to Enterprise accounts.

Reply:
That doesn't quite work since it doesn't direct them to contact their TAMs

How bout: 
```
<banner status="info"><a href="https://www.sparkpost.com/enterprise-email/"><span class="label label-warning"><strong>Enterprise</strong></span></a> accounts, contact your TAM for details.</banner>
```